build-tools/src/main/resources/build-tools/spotbugs-filter.xml

```diff
@@ -71,4 +71,9 @@ limitations under the License.
     <Bug pattern="NM_SAME_SIMPLE_NAME_AS_SUPERCLASS" />
   </Match>
 
+  <Match>
+    <Class name="org.apache.knox.gateway.topology.discovery.cm.ClouderaManagerClusterConfigurationMonitor" />
+    <Bug pattern="OBJECT_DESERIALIZATION" />
```

Comment:
This is super concerning. If this is required (I'm not sure it is based on looking at this class), can this be limited to a single method instead of the whole class?